src/api/ZoweExplorerApi.ts

```diff
@@ -106,6 +107,20 @@ export namespace ZoweExplorerApi {
             returnEtag?: boolean
         ): Promise<zowe.IZosFilesResponse>;
 
+        /**
+         * Uploads the file at the given path. Use for Save.
+         *
+         * @param {string} inputFilePath
+         * @param {string} ussFilePath
+         * @param {zowe.IUploadOptions} [options]
+         * @returns {Promise<zowe.IZosFilesResponse>}
+         */
+        putContent?(
+            inputFilePath: string,
+            ussFilePath: string,
+            options?: zowe.IUploadOptions
+        ): Promise<zowe.IZosFilesResponse>;
```

Comment:
Interesting why we never thought of this since we did it the right way for `getContents`
Now we have a deprecated `putContents` and regular `getContents`.
Either way, this serves as a great example for how we will be enhancing the APIs (that we plan to deliver as a separate package)

@phaumer, @lauren-li, Check this out! 😋

Reply:
Yes, it would be nice for RSE and FTP extensions to implement `putContent` soon.